### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Note:  filters are not implemented as of now.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import "./App.css";
 import { Footer, Navbar, Homepage } from "./Components/Allcomps";
+import { ProductsPage } from "./Components/Products/ProductsPage";
```
Comment (Shubhamkr2610) [inline src/App.js]: Appreciation: Its great to see that you are using absolute import here . I also use this way to import the files

File: src/data/Data.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+import { ArmaniAnalogWatch, ArmaniChronoWatch, BovetAutomaticWatch, CasioChronoWatch, HrxSmartWatch, MiSmartWatch, OnePlusSmartWatch, RadoAnalogWatch, RiderAnalogWatch, SeikoAutomaticWatch} from "../images/ImageExp"
+
```
Comment (Shubhamkr2610) [inline src/data/Data.js:2]: Appreciation: great logic . it reduced your also almost 10-12 lines of code , looking clean too , improving readability

**Review by Shubhamkr2610 — APPROVED** [review]
Everything looks good to me , your logic of importing files is good. you can merge it

File: src/Components/Products/ProductsPage.jsx [hunk 3]
```diff
@@ -11,6 +11,23 @@ const ProductsPage = () => {
...
+            <span><input className="checkbox" type="checkbox" />Price: Low to high</span>
+            <span><input className="checkbox" type="checkbox" /> Rating</span>
+         
+
+          <h3>by category</h3>
+            <span><input className='radio' type="radio" name="" id="" />Analog</span>
+            <span><input type="radio" name="" id="" />Smart</span>
+            <span><input type="radio" name="" id="" />Automatic</span>
+            <span><input type="radio" name="" id="" />Chronograph</span>
+            </div>
```
Comment (Shubhamkr2610) [inline src/Components/Products/ProductsPage.jsx:28]: suggestion: if you are not using id and name in input tag  , you can remove it , unnecessarily not needed to put in code

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Muzammil-khan710) [reply src/App.js]: Thanks @Shubhamkr2610

File: src/data/Data.js (hunk 2 above)
Comment (Muzammil-khan710) [reply src/data/Data.js:2]: Yeah it really saves line of codes

**Review by tonymike99 — APPROVED** [review]
Hi Muzammil :)
No problems with the code. You can proceed with the merge.

**Pull request merged**